flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/rules/datastream/DataStreamOverAggregateRule.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.plan.rules.datastream
+
+import org.apache.calcite.plan.volcano.RelSubset
+import org.apache.calcite.plan.{Convention, RelOptRule, RelTraitSet}
+import org.apache.calcite.rel.RelNode
+import org.apache.calcite.rel.convert.ConverterRule
+import org.apache.calcite.rel.logical.LogicalWindow
+import org.apache.flink.table.calcite.FlinkRelBuilder.NamedWindowProperty
+import org.apache.flink.table.plan.nodes.datastream.DataStreamConvention
+import org.apache.flink.table.plan.nodes.datastream.DataStreamOverAggregate
+
+import scala.collection.JavaConversions._
+
+/**
+  * Rule to convert a LogicalWindow into a DataStreamOverAggregate.
+  */
+class DataStreamOverAggregateRule
+  extends ConverterRule(
+    classOf[LogicalWindow],
+    Convention.NONE,
+    DataStreamConvention.INSTANCE,
+    "DataStreamOverAggregateRule") {
+
+  override def convert(rel: RelNode): RelNode = {
```

Comment:
When I have rebased the PR on the latest master and try to use `ProcTime()/RowTime()`, I got a exception:
`org.apache.flink.table.codegen.CodeGenException: Unsupported call: PROCTIME 
If you think this function should be supported, you can create an issue and start a discussion for it.atorg.apache.flink.table.codegen.CodeGenerator$$anonfun$visitCall$3.apply(CodeGenerator.scala:1116)`
The reason for the exception is that "EventTimeExtractor / ProcTimeExtractor" is not processed in "FunctionGenerator # getCallGenerator". Do we open a new JIRA for processing or fix it in the current PR? What do you think?